app/services/hmpps_api/api_deserialiser.rb

```diff
@@ -9,13 +9,5 @@ def deserialise_many(memory_model_class, payload_list)
         }
```

Comment:
Is there any value in also losing the `ApiDeserialiser#deserialise_many` method? And then we can get rid of the whole class.